core/src/testils/proptest/mod.rs

```diff
@@ -16,12 +16,12 @@
 * SPDX-License-Identifier: Apache-2.0
 */
 
-//! Defines extended [`proptest`] [`Arbitrary`] traits and [`Strategies`] for
+//! Defines extended [proptest] [Arbitrary] traits and [Strategies] for
```

Comment:
Arbitrary and Strategy are both traits. Maybe change Strategies to singular? Something like 

> Defines extended proptest `Arbitrary` and `Strategy` traits...